social/src/Services/UpdateUser.js

```diff
@@ -0,0 +1,17 @@
+import { db } from '../firebase';
+import { doc, updateDoc } from "firebase/firestore";
+import { showToast, isLoading } from '../Features';
+import { getUser } from '.';
+
+export const UpdateUser = async(form, dispatch, token) => {                          
+    dispatch(isLoading(true))
+  try {
+    await updateDoc(doc(db, "users", token), form);
```

Comment:
use proper variable name for the `form` variable like, `updatedUserData` etc.

Reply:
Okay will change this